src/edu/pcc/cis/conversion/ConversionFunctions.java

```diff
@@ -5,6 +5,7 @@
  *
  * @author Charles Leonard (@orig Cara Tang)
  * @version 2018.05.01
+ * I may not have done the lab in exactly the order proscribed.
  */
```

Comment:
Looks like you initialized the repo with most of the solution code instead of doing it on a branch to be part of the pull request.